### PR TITLE
Minitest 5

### PR DESCRIPTION
**Pull request by shepmaster**

### CODE REVIEW

File: acceptance/minitest_example_test.rb [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+
+class ExampleWithAnError < Minitest::Test
+  def test_error
     raise "second failure"
   end
 end
 
-class MiniTestExampleTestTwo < MiniTest::Unit::TestCase
+class ExampleThatPasses < Minitest::Test
   def test_two
```
Comment (carols10cents) [inline acceptance/minitest_example_test.rb]: I would have expected you to rename this test to `test_passes` :)

File: acceptance/minitest_example_test.rb [hunk 2]
```diff
@@ -1,17 +1,29 @@
 require 'minitest/autorun'
 
-class MiniTestExampleTestOne < MiniTest::Unit::TestCase
-  def test_one
-    puts "Some <![CDATA[on stdout]]>"
```
Comment (carols10cents) [inline acceptance/minitest_example_test.rb:5]: Was this previously testing (implicitly) that `<`s are escaped correctly when stdout is put into the XML, and have you lost that coverage now?

Comment (shepmaster) [inline acceptance/minitest_example_test.rb:5]: Yes, but this is a very directed test for a class that isn't in this repo (it's in ci_reporter proper). I'll add unit tests over there.

File: acceptance/minitest_example_test.rb (hunk 1 above)
Comment (shepmaster) [inline acceptance/minitest_example_test.rb]: Done

**Pull request merged**